pkg/monitors/kubernetes/cluster/metrics/containers.go

```diff
@@ -37,7 +37,7 @@ func datapointsForContainerSpec(c v1.Container, contDims map[string]string) []*d
 			datapoint.New(
 				"kubernetes.container_cpu_request",
 				contDims,
-				datapoint.NewIntValue(val.Value()),
```

Comment:
Really it should be

`datapoint.NewFloatValue(float64(val.MilliValue())/1000.0)`

But I'll test and update this PR once I've verified that that works.

Thanks for bringing this up.

Reply:
I'm fine with either that `datapoint.NewFloatValue(float64(val.MilliValue())/1000.0)` or `datapoint.NewIntValue(val.MilliValue())`.